openedx/core/djangoapps/api_admin/migrations/0002_auto_20160325_1419.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def add_api_access_group(apps, schema_editor):
+    Group = apps.get_model('auth', 'Group')
+    Permission = apps.get_model('auth', 'Permission')
+    ContentType = apps.get_model('contenttypes', 'ContentType')
+    ApiAccessRequest = apps.get_model('api_admin', 'ApiAccessRequest')
+
+    group, __ = Group.objects.get_or_create(name='api_access_request_approvers')
+    api_content_type = ContentType.objects.get_for_model(ApiAccessRequest)
+    group.permissions = Permission.objects.filter(content_type=api_content_type)
+    group.save()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api_admin', '0001_initial'),
+        ('contenttypes', '0002_remove_content_type_name')
+    ]
+
+    operations = [
+        migrations.RunPython(add_api_access_group)
```

Comment:
That was related to users. I'm not certain a consensus was actually reached. I honestly found it all a bit overkill. I'm betting 99.99% of OpenEdX installations do not have a group matching either of the proposed names.

We could safely avoid issues with the 0.01% by using `create` instead of `get_or_create`, forcing those users to fork and change the group name. You could also add yet another setting that almost everyone will ignore. The choice is yours.